src/index.html

```diff
@@ -120,15 +120,22 @@ <h1 class="error-title js-error-title"></h1>
   <script src="./scripts/push-client.js"></script>
   <script src="./scripts/app-controller.js"></script>
   <script src="./scripts/main.js"></script>
-
+  <script src="./scripts/libs/idb-keyval.js"></script>
   <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
   })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
   ga('create', 'UA-77119321-2', 'auto');
   ga('send', 'pageview');
+  ga(function(tracker) {
+    var clientId = tracker.get('clientId');
+    idbKeyval.set('google-analytics-client-id', clientId)
+    .catch(function(err) {
+      console.error('Unable to track Google Analytics with Client ID.');
```

Comment:
This message probably shouldn't sound so scary. If the client ID fails to get written to IndexedDB, normal tracking will still work.